docs/stories/components/charts/LChartGrid.stories.js

```diff
@@ -0,0 +1,31 @@
+import LChartGrid from '~/src/components/charts/LChartGrid.vue';
+
+export default {
+  title: 'Components/Charts/ChartGrid',
+  component: LChartGrid,
+  argTypes: {
+      data: { control: 'array', description: 'Array of objects for each item on the chart. Items must have: title, description and imagePath' },
+      headers: { control: 'array', description: 'Array of objects for each item on the chart. Items must have: title, description and imagePath' }
+  }
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { LChartGrid },
+  template: '<l-chart-grid v-bind="$props"/>'
```

Comment:
```suggestion
  template: `
    <l-chart-grid v-bind="$props" />
  `
```
Veja se o github não cagou a identação

Reply:
arrumado!